huaweicloud/resource_huaweicloud_cce_addon_v3.go

```diff
@@ -0,0 +1,205 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/cce/v3/addons"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCCEAddonV3() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCCEAddonV3Create,
+		Read:   resourceCCEAddonV3Read,
+		Delete: resourceCCEAddonV3Delete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(3 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{ // request and response parameters
+			"authenticating_proxy_ca": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"addon_version": {
```

Comment:
better to name this to just `version` instead of `addon_version`, as it's in the addon resource, no need to mention that again.